static/chrome/entrypoint.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash
 
 if [ "$(id -u)" = "0" ]; then
-    # Fix hostname resolution to avoid DNS timeout (needs root)
-    echo "127.0.0.1 $HOSTNAME" >> /etc/hosts
```

Comment:
In previous pr I add this in order to fix problem with `GetCanonicalHostName Error hostname`. But the problem was in used `--whitelisted-ips=''`. In new chromedriver need to use: `--allowed-ips` and `--allowed-origins` which are already used. So I remove this code.